subprojects/android/config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+<module name="Checker"/>
```

Comment:
@mureinik, do we need separate checker configurations per submodule? I'd rather us have a single consistent checkstyle rule for all java code.